proto/mixer.proto

```diff
@@ -190,13 +190,15 @@ message ObsTimeSeries {
     string measurement_method = 3;
     string observation_period = 4;
     string import_name = 5;
+    string provenanceDomain = 6;
   }
   map<string, double> data = 1;  // Date to value.
   string unit = 2;
   string place_name = 3;
   bool is_dc_aggregate = 4;
   string place_dcid = 5;  // Only set if the key uses mid.
   repeated SourceSeries source_series = 6;
+  string provenanceDomain = 7;
```

Comment:
Puzzled why this is needed since you have it in SourceSeries?

Reply:
Now i only return one source series, and this is popped to upper level.

Reply:
Is it temporary then? Add this comment to proto and have a TODO to remove if temp (later PR is fine).